x-pack/plugin/src/yamlRestTestV7Compat/resources/rest-api-spec/test/freeze.gone/10_basic_compat.yml

```diff
@@ -1,9 +1,10 @@
 ---
 setup:
   - skip:
-      version: "9.0.0 - "
+      cluster_features: [] # use 9.x feature to skip
```

Comment:
These 2 are probably useless: we will not run  v7 compat with v9. I think the skip statement can be removed altogether

Reply:
Removed 👍